cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesGroupController.java

```diff
@@ -239,31 +239,31 @@ public void create(@NotNull Context ctx) {
                 + "unassign all existing time series before assigning new time series specified in the content body "
                 + "Default: false"),
             @OpenApiParam(name = OFFICE, required = true, description = "Specifies the "
-                + "owning office of the time series group to be updated"),
+                + "user office making the request"),
```

Comment:
same comment as the other controller

Reply:
I expanded the description a bit to be more representative of the actual use of the parameter